src/ShopBundle/Resources/translations/messages.en.xliff

```diff
@@ -83,15 +83,15 @@
             </trans-unit>
             <trans-unit id="chameleon_system_shop.payment_data_trans.form_card_valid_until_help">
                 <source>chameleon_system_shop.payment_data_trans.form_card_valid_until_help</source>
-                <target>This date you'll find at the front of your credit card</target>
+                <target>This date can be found at the front of your credit card.</target>
```

Comment:
shouldn't it be "on the front side" instead of "at the front"?

Reply:
Changed